projects/vdk-plugins/vdk-impala/src/vdk/plugin/impala/templates/load/dimension/scd1/02-handle-quality-checks.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import re
+
+from vdk.api.job_input import IJobInput
+
+SQL_FILES_FOLDER = (
+    os.path.dirname(os.path.abspath(__file__)) + "/02-requisite-sql-scripts"
+)
+
+
+def run(job_input: IJobInput):
+    job_arguments = job_input.get_arguments()
+
+    check = job_arguments.get("check")
+    target_schema = job_arguments.get("target_schema")
+    target_table = job_arguments.get("target_table")
+    staging_schema = job_arguments.get("staging_schema")
```

Comment:
Let's have a default as well. If not specified uses same as target_schema 
```suggestion
    staging_schema = job_arguments.get("staging_schema". target_schema)
```

Reply:
Wouldn't this cause an error? In the current logic the table name will be the same for both target and staging schemas.

Reply:
That's true. I didn't realize. 

But it's better UX to require as few users as possible to get started with. This means as many as possible parameters to have defaults.
Wouldn't be better to prefix the table as well? If one uses a staging db for multiple purposes, this way it'd be easy to see those tables are generated for checks. Maybe we can add prefix "vdk_check_" ? 

What do you think ?